webui.py

```
@@ -47,17 +47,42 @@
 # Create the global agent state instance
 _global_agent_state = AgentState()
 
+def resolve_sensitive_env_variables(text):
+    """
+    Replace environment variable placeholders ($SENSITIVE_*) with their values.
+    Only replaces variables that start with SENSITIVE_.
+    """
+    if not text:
+        return text
+        
+    import re
```

Ruff (0.8.2)

Redefinition of unused `re` from line 12

Remove definition: `re`

(F811)